easybuild/easyconfigs/c/CUDA/CUDA-9.0.176-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,22 @@
+name = 'CUDA'
+version = '9.0.176'
+nv_version = '384.81'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+source_urls = [
+    'https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/prod/local_installers/',
+    'http://developer.download.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/',
+    'https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/Prod2/local_installers/',
+    'https://developer.nvidia.com/compute/%(namelower)s/%(version_major_minor)s/Prod/local_installers/',
+]
+sources = ['%%(namelower)s_%%(version)s_%s_linux-run' % nv_version]
+#checksums = ['8496c72b16fee61889f9281449b5d633d0b358b46579175c275d85c9205fe953']
```

Comment:
@akesandgren Please fix? I have `96863423feaa50b5c1c5e1b9ec537ef7ba77576a3986652351ae43e66bcd080c` as SHA256 for `cuda_9.0.176_384.81_linux-run`, this one you have here is the one for `cuda_9.1.85_387.26_linux`

Reply:
Oops